CSS_files/components/image/image.css

```diff
@@ -1,8 +1,20 @@
-img{
+.resp-img{
    width: 100%;
+   height: auto;
    object-fit: contain; 
 }
 
 .img-circle{
```

Comment:
Can be consistent with class name by using `circle-img` or `round-img` as `resp-img`

Reply:
I have made the necessary changes, thank you for the feedback.